### DISCUSSION TITLE
Hardware Requirements Removed!

### DISCUSSION THREAD

**Discussion by DCullenV** [Zer0-bit/gaggiuino#217] [General]
I noticed that in the last couple of days the Hardware Requirements section was removed from the current documentation for both the Gaggia Classic and Gaggia Classic Pro.

Was this intentional?

I found it more difficult to decide what parts I need to obtain for my Gaggiuino as there are some differences between the parts still listed in BOM and those that were previously listed in the sections that were removed.

**Comment by Zer0-bit** [comment]
It's not been removed but moved to it's own section to aggregate both versions in a single list.
